Hazel/src/Hazel/Core/MouseButtonCodes.h

```diff
@@ -1,6 +1,32 @@
 #pragma once
 
-// From glfw3.h
+namespace Hazel
+{
+    enum class MouseButtons
```

Comment:
Maybe rename this so `MouseButtons::Button1` becomes `Mouse::Button1` to be inline with the key enum.
```suggestion
    enum class Mouse
```

_Here there is no typedef required as we only use 1 mouse._ ;)